.github/workflows/poetry-autoupdate.yml

```
@@ -0,0 +1,39 @@
+---
+name: Check for poetry updates and open PR
+on:
+  workflow_dispatch: ~
+  schedule:
+    - cron: '0 6 * * *'
+
+jobs:
+  update-dep:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Update poetry to latest version.
+        run: |
+          VERSION="$(curl -f -s -H "Accept: application/json" "https://api.github.com/repos/python-poetry/poetry/releases/latest" | sed -n '/tag_name/p' | cut -d: -f2 | sed 's/[",[:space:]]//g')"
```

This could maybe be replaced with https://github.com/marketplace/actions/github-api-request

---

Using `jq` could work too and would be more reliable I think.

```suggestion
          VERSION="$(curl -f -s -H "Accept: application/json" "https://api.github.com/repos/python-poetry/poetry/releases/latest" | jq -r '.tag_name')"
```

---

As far as I know jq is not installed by default. I will add a step to install it and update this.

---

Add now.